### DISCUSSION TITLE
Profile topics

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#86573] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice if the topics from the [profile README repository](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme) would be shown on the profile:
![grafik](https://github.com/community/community/assets/11225821/79c1d85f-4e5f-4ed7-9237-e3dee90ddb14)

To see a more in-depth list of a user's topics, one could look at a dedicated tab:
![grafik](https://github.com/community/community/assets/11225821/ad6c9ed2-1496-4b03-a59e-c6a9421ae429)

![grafik](https://github.com/community/community/assets/11225821/de6a67fc-e5a9-4d5f-ac89-d33479095c4d)

**Comment by dariohsb** [comment]
Hello mate, that's a great idea! Assuming your post doesn't require a specific answer, could you please mark my response as the answer? Thank you.

**Reply by shaedrich** [reply]
Why yours in particular?

Besides, the answer that _would_ apply would be one of a GitHub staff member announcing the implementation of what I suggested in some shape or form.
